start.js

```diff
@@ -17,6 +17,21 @@ if (!envExists) {
   .pipe(fs.createWriteStream(path.join(__dirname, '/.env')))
 }
 
+// Create template session data defaults file if it doesn't exist
+const sessionDefaultsDirectory = path.join(__dirname, '/app/data')
+const sessionDefaultsFile = path.join(sessionDefaultsDirectory, '/session-defaults.js')
+const sessionDefaultsExists = fs.existsSync(sessionDefaultsFile)
+
+if (!sessionDefaultsExists) {
+  console.log('Creating template session data defaults file')
```

Comment:
Are we creating a template file, or just the defaults file?

Reply:
I agree, we should strip "template" from the sentence as we're creating from a template, not creating a template